adapters/github.md

```diff
@@ -0,0 +1,672 @@
+# Github CloudEvents Adapter
+
+This document describes how to convert
+[Github webhook events](https://developer.github.com/v3/activity/events/types/)
+into a CloudEvents.
+
+Github webhook event documentation:
+https://developer.github.com/v3/activity/events/types/
+
+Each section below describes how to determine the CloudEvents attributes
+based on the specified event.
+
+### CheckRunEvent
+
+| CloudEvents Attribute | Value                                                                            |
+| :-------------------- | :------------------------------------------------------------------------------- |
+| `id`                  | "X-GitHub-Delivery" HTTP header value                                            |
+| `source`              | "repository.url" value                                                           |
+| `specversion`         | `0.3-wip`                                                                        |
+| `type`                | `github.com/check_run.` + "action" value                                         |
+| `datacontentencoding` | Omit                                                                             |
+| `datacontenttype`     | `application/json`                                                               |
+| `schemaurl`           | Omit                                                                             |
```

Comment:
it would be great to have an example that uses `schemaurl`

Reply:
yep